scripts/new_project.kts

```diff
@@ -225,6 +235,12 @@ object NewProject {
         println("\n${message}\n")
         System.exit(exitCode)
     }
+
+    private fun String.uppercaseEveryFirstWord(): String {
```

Comment:
How about renaming this to be more accurate? 😄 
```suggestion
    private fun String.uppercaseEveryFirstCharacter(): String {
```
```suggestion
    private fun String.titlecase(): String {
```

Reply:
`uppercaseEveryFirstCharacter` seems good to me. Will update it @ryan-conway